docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1595,7 +1600,12 @@ through the shared lib at `wlp/user/shared/resources/lib`) |
 |`fhirServer/core/conditionalDeleteMaxNumber`|10|
 |`fhirServer/core/capabilityStatementCacheTimeout`|60|
 |`fhirServer/core/extendedCodeableConceptValidation`|true|
-|`fhirServer/searchParameterFilter`|`"*": [*]`|
+|`fhirServer/resources/open`|true|
+|`fhirServer/resources/Resource/interactions`|null (all interactions supported)|
+|`fhirServer/resources/Resource/searchParameters`|null (all global search parameters supported)|
+|`fhirServer/resources/<resourceType>/interactions`|null (inherets from `fhirServer/resources/Resource/interactions`)|
+|`fhirServer/resources/<resourceType>/searchParameters`|null (all type-specific search parameters supported)|
+|`fhirServer/resources/<resourceType>/searchParameters/<code>/url`|string|null|
```

Comment:
```suggestion
|`fhirServer/resources/<resourceType>/searchParameters/<code>/url`|string or null|
```

should the pipe be considered an or ? the MD considers it a column delimiter

Reply:
copy -paste error from the previous section.  fixed by dropping the middle "column"